src/ui/profile_preferences.ui

```diff
@@ -2067,6 +2041,113 @@
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="tab_logging">
+      <attribute name="title">
+       <string>Logging</string>
+      </attribute>
+      <layout class="QGridLayout" name="gridLayout_30">
+       <item row="0" column="0">
+        <widget class="QGroupBox" name="groupBox_logFiles">
+         <property name="title">
+          <string>Log files</string>
+         </property>
+         <layout class="QGridLayout" name="gridLayout_31">
+          <item row="1" column="0">
```

Comment:
Any time you mess with a `QGridLayout` in a form/dialogue I strongly advise you to go through the XML `.ui` file with the **Plain Text Editor** afterwards and sort and reorder the `<item `....`>`...`</item>` entries at a particular indentation (level) in the file into ascending row (outer sort) and then column order (inner sort) - in the editor you can collapse/fold each `<item `....`>`...`</item>` down and then select the whole entry and cut and paste it manually in the right place with the right indentation (except if it becomes the **first** element - that one gets messed up by such an action so if moving that one put it second and then cut the very first one above it then put that one below it...)  The Qt Designer code is buggy in all the Qt Creator versions I have seen and it does not retain the order and it can also introduce unused values in the row or column sequence so that it is not monotonic {goes 0,1,2,3,4,5,6,7,8 etc...} - **however that makes the changes in git much larger and much more difficult for humans to parse the actual changes being made...** :neckbeard: